common-bsp/recipes-kernel/device-tree-overlays/device-tree-overlays_git.bb

```diff
@@ -0,0 +1,33 @@
+DESCRIPTION = "Device Tree Overlays for bb.org boards"
+
+LICENSE = "MIT"
```

Comment:
Upstream bb.org-overlays repository does not include any licensing information, so I thought it would be best to match meta-beagleboard license.

I'd be happy to change this to whatever you prefer.